README.md

```diff
@@ -1,3 +1,4 @@
 # TicTacToe_Ruby_v2
 
-[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=master)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
+
+[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=convert_to_gem)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
```

Comment:
yeah, I didn't think about that. How would you suggest in the future having a way of showing the current build branch in the README but not having that merge into the master when a pull is made?